pkgs/servers/monitoring/matrix-alertmanager/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, callPackage, mkYarnPackage, fetchYarnDeps, fetchFromGitLab, nodejs }:
+
+mkYarnPackage rec {
+  pname = "matrix-alertmanager";
+  version = "0.4.0";
+
+  src = fetchFromGitLab {
+    domain = "git.feneas.org";
+    owner = "jaywink";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1q9vrklcnlmcgj4mms3im54zbwjjb2k1q0883v8gxl8hzqykl315";
+  };
+
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+
+  offlineCache = fetchYarnDeps {
+    inherit yarnLock;
+    sha256 = lib.fileContents ./yarn-hash;
```

```suggestion
    sha256 = "1yvcm0prk8lxljlnww42pdgzdjdhljkih94hqb0x4cikd2cqgjmp"
```
it should be pretty easy to write a sed to inject that. Do you need help with that?

---

I don't see any advantage in doing it that way